runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectRunner.java

```diff
@@ -274,12 +268,12 @@ public MetricResults metrics() {
     }
 
     /**
-     * Blocks until the {@link Pipeline} execution represented by this
-     * {@link DirectPipelineResult} is complete, returning the terminal state.
+     * Blocks until the {@link Pipeline} execution represented by this {@link DirectPipelineResult}
```

Comment:
I think the original interface docs suffice here, don't they? Incidentally, can this class be private or package-private?